pkg/network/telemetry/stat_gauge_wrapper.go

```diff
@@ -0,0 +1,49 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package telemetry
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/telemetry"
+	"go.uber.org/atomic"
+)
+
+type StatGaugeWrapper struct {
```

Comment:
I would have thought this would be exactly like `telemetry.Gauge`. The tags parameters are important. You can not use the tags for the `stat` field and just pass them to the `gauge` operation.

Reply:
What is the purpose of this wrapper?

Reply:
the wrapper is just a convenience type for telemetry we want to add prometheus metrics for while continuing to return as expvars

Reply:
Is there a reason to keep exporting them as expvars?